geometry_msgs/PoseArray waypoints
---
bool collision